README.md

```diff
@@ -56,39 +64,125 @@ Morpheus Feedback API
 - OpenShift (for production deployment)
 
 ---
+
 ## Installation and Setup
 
 1. Clone the Repository:
-    ```
-    git clone https://github.com/your-repo/morpheus-feedback-api.git
-    cd morpheus-feedback-api
-    ```
+   ```
+   git clone https://github.com/your-repo/morpheus-feedback-api.git
+   cd morpheus-feedback-api
+   ```
 2. Create a Virtual Environment:
-    ```
-    python -m venv venv
-    source venv/bin/activate
-    ```
+
+   ```
+   python -m venv venv
+   source venv/bin/activate
+
+   ```
 3. Install Dependencies:
-    ```
-    pip install -r requirements.txt
-    ```
-4. Set Up Configuration:
-
-    Update config.py file with the following variables:
-    ```
-    ARGILLA_API_URL=<your-argilla-api-url>
-    ARGILLA_API_KEY=<your-argilla-api-key>
-    ARGILLA_DATASET  = <your-argilla-dataset>
-    ARGILLA_WORKSPACE = <your-argilla-workspace>
-    ```
+
+   ```bash
+   pip install -r requirements.txt
+   ```
+
+ **Note:** The configuration is set with default values, which can be viewed in the configuration file.
```

Comment:
please provide the user with a link or path to the configuration file so that the user can locate it within the codebase